staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -1005,3 +1010,16 @@ type ExecEnvVar struct {
 	Name  string `json:"name"`
 	Value string `json:"value"`
 }
+
+// RateLimitConfiguration is used for setting the rate limits for the kubelet APIs exposed
+// locally on the node as gRPC over UNIX-domain sockets, like the podresources API
+type RateLimitConfiguration struct {
+	// MaxFrequency  defines the maximum allowed frequency of the calls represented
+	// as number of calls per second, cumulating all the calls.
+	// Setting the values to zero means no limit.
+	MaxFrequency float64 `json:"maxFrequency,omitempty"`
+	// MaxBurst defines the maximum numbers of calls that may happen at once, within
+	// the boundaries set by frequency.
+	// This value is ignored if the frequuency is unlimited.
```

Comment:
need to define default behavior if unset/zero

```suggestion
	// This value is ignored if the frequency is unlimited.
```